index.html

```diff
@@ -0,0 +1,720 @@
+        </ul>
+    </nav>
+    <div class="flex nav-2">
+        <div class="nav-2-left-holder">
+            <img class="hidden-1 nav-img" id="overwrite" alt="The Next Web"
+                src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg">
+            <ul class="flex">
+                <li class="color-y">LATEST</li>
+                <li class="color-y">HARD FORK</li>
+                <li class="color-y">PLUGGED</li>
+                <li class="color-y">README</li>
+                <li class="color-y">GROWTH QUARTERS</li>
+                <li class="color-y">NEURAL</li>
+            </ul>
+        </div>
+        <div class="flex nav-2-right">
+            <i class="fas fa-search fa-flip-horizontal nav-2-icon"></i>
+            <i class="fas fa-bars nav-2-icon-2 hidden-2" id="hidden-2"></i>
+        </div>
+    </div>
+
+    <div class="top-grid">
+        <div class="top-grid-1">
+            <div class="child-sec">
+                <div class="head-text">
+                    <div class="logo-date">
+                        <div class="tmw-logo"><img src="./images/tnw.svg" alt="logo"></div>
+                    </div>
+                    <p class="current-date white-color">Wednesday - 5, August 2020</p>
+                    <div class="move-content">
+                        <div class="headings">INSIGHTS</div>
+                        <div class="head-text-title">Kodak, now a pharma company, expects drugs to make up for 30-40% of
+                            its
+                            future business.</div>
+                        <div class="head-text-bottom">by Ivan Mehta <i class="far fa-clock"></i> 1 hour ago <i
+                                class="fas fa-share-alt"></i> 6</div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="side-sec">
+            <div class="top-grid-2">
+                <div class="child-sec">
+                    <div class="head-text">
+                        <div class="headings">TECH</div>
+                        <div class="headlines">CES Officially goes digital in 2021.
+                        </div>
+                        <div class="head-text-bottom">by Rachel <i class="far fa-clock"></i> 6 houra ago <i
+                                class="fas fa-share-alt"></i> 10</div>
+                    </div>
+                </div>
+            </div>
+            <div class="top-grid-3">
+                <div class="child-sec">
+                    <div class="head-text">
+                        <div class="headings">APPS</div>
+                        <div class="headlines">This spooky Deepfake AI mimics dozens of celebs.</div>
+                        <div class="head-text-bottom">by Napier Lopez <i class="far fa-clock"></i> 7 houra ago <i
+                                class="fas fa-share-alt"></i>
+                            11</div>
+                    </div>
+                </div>
```

Comment:
- [ ] this content needs to be centralized (in the mobile screen size)

![image](https://user-images.githubusercontent.com/61479648/89557982-02f2b100-d7ea-11ea-831d-114e238bd52c.png)